### PR TITLE
[4.0] namespace administrator components com_config

### PR DESCRIPTION
**Pull request by brianteeman**
continuing the work to namespace joomla4 one admin component at a time.

please wait for drone to successfully complete before reviewing.

### CODE REVIEW

File: administrator/components/com_config/Field/ConfigComponentsField.php [hunk 1]
```diff
@@ -12,6 +12,9 @@
 defined('JPATH_BASE') or die;
 
 use Joomla\Utilities\ArrayHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Fileystem\File;
```
Comment (Quy) [inline administrator/components/com_config/Field/ConfigComponentsField.php]: Spelling `Fileystem`

File: administrator/components/com_config/Model/ApplicationModel.php [hunk 2]
```diff
@@ -11,14 +11,26 @@
...
 use Joomla\CMS\Table\Asset;
 use Joomla\CMS\Table\Table;
 use Joomla\Registry\Registry;
 use Joomla\Utilities\ArrayHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\User\UserHelper;
+use Joomla\CMS\Client\ClientHelper;
+use Joomla\CMS\Cache\Exception\CacheConnectingException;
+use Joomla\CMS\Cache\Exception\UnsupportedCacheException;
+use Joomla\CMS\Fileystem\File;
```
Comment (Quy) [inline administrator/components/com_config/Model/ApplicationModel.php]: Spelling `Fileystem`

**Comment by wilsonge** [comment]
Thanks guys!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**